src/components/__tests__/BannerRoutes.cy.js

```diff
@@ -40,18 +45,103 @@ describe('<BannerRoutes>', () => {
         .should('have.css', 'text-transform', 'uppercase')
         .should('have.color', '#fff')
         .should('have.css', 'border-radius', '28px')
+        .should('have.css', 'padding-top', '16px')
+        .should('have.css', 'padding-left', '16px')
+        .should('have.css', 'padding-bottom', '16px')
+        .should('have.css', 'padding-right', '24px')
         .should('contain', i18n.global.t('index.bannerRoutes.addRoutes'));
     });
 
-    it('renders title icon', () => {
+    it('renders button icon with correct spacing', () => {
+      cy.dataCy('banner-routes-button-icon')
+        .should('be.visible')
+        .should('have.color', '#fff')
+        .should('have.css', 'margin-right', '8px')
+        .should('contain', 'add');
+      cy.dataCy('banner-routes-button-icon')
+        .invoke('height')
+        .should('equal', 24);
+      cy.dataCy('banner-routes-button-icon')
+        .invoke('width')
+        .should('equal', 24);
+    })
+
+    it('has gray background', () => {
+      cy.window().then(() => {
+        cy.dataCy('banner-routes-card')
+          .should('be.visible')
+          .should('have.backgroundColor', '#fafafa'); // grey-1
```

Comment:
Better to use Quasar framework `getPaletteColor()` func util:

```ts
import { colors } from 'quasar';

import BannerRoutes from 'components/BannerRoutes.vue';
import { i18n } from '../../boot/i18n';

const { getPaletteColor } = colors;
```

See  `getPaletteColor()` func [doc](https://quasar.dev/quasar-utils/color-utils#helper-getpalettecolor):

```ts
 .should('have.backgroundColor', getPaletteColor('grey-1'));
```

Reply:
Please change it in other places as well.